### PR TITLE
Fix bogus dbinfo message

### PR DESCRIPTION
**Pull request by akshatsikarwar**
/silent

### CODE REVIEW

File: plugins/newsql/newsql_evbuffer.c [hunk 1]
```diff
@@ -356,10 +357,23 @@ static void process_dbinfo_int(struct newsql_appdata_evbuffer *appdata, struct e
     struct newsqlheader hdr = {0};
     hdr.type = htonl(RESPONSE_HEADER__DBINFO_RESPONSE);
     hdr.length = htonl(len);
-    uint8_t out[len];
+    uint8_t *out = malloc(len);
     cdb2__dbinforesponse__pack(&response, out);
     evbuffer_add(buf, &hdr, sizeof(hdr));
     evbuffer_add(buf, out, len);
```
Comment (chands10) [inline plugins/newsql/newsql_evbuffer.c:363]: Should we still send if can't decode?

Comment (mponomar) [reply plugins/newsql/newsql_evbuffer.c:363]: Where does it get freed?

Comment (akshatsikarwar) [reply plugins/newsql/newsql_evbuffer.c:363]: What would that do? Client will fail to decode and crash.

Comment (chands10) [reply plugins/newsql/newsql_evbuffer.c:363]: Right now it looks like the code can potentially send the message first and then crash

Comment (chands10) [reply plugins/newsql/newsql_evbuffer.c:363]: Ah ok ic

**Review by chands10 — DISMISSED** [review]

File: plugins/newsql/newsql_evbuffer.c (hunk 1 above)
Comment (akshatsikarwar) [reply plugins/newsql/newsql_evbuffer.c:363]: We write after this routine returns (which we won't, if our buffer is trashed):
https://github.com/bloomberg/comdb2/blob/master/plugins/newsql/newsql_evbuffer.c#L367-L368

Comment (chands10) [reply plugins/newsql/newsql_evbuffer.c:363]: Ah I was thinking about bufferevents which I guess write for you

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Error &#x26A0;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>236/470 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>insert_lots [setup failure]<br>idxexpr_alltypes [setup failure]<br>tmptbl_deadlock_starve_generated [setup failure]<br>selectv_rcode_force_commit_generated [setup failure]<br>ruleset_no_fp_generated [setup failure]<br>cldeadlock<br>sc_truncate_lockorder_generated<br>sc_truncate<br>logfill<br>delfile_block_reproleak_generated</p>

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Error &#x26A0;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>195/470 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>limitsortingtbl [setup failure]<br>sc_addfield_logicalsc_generated [setup failure]<br>tmptbl_deadlock_bigpool_generated [setup failure]<br>selectv_recom_range [setup failure]<br>sc_newuniq [setup failure]<br>drop_tblnum [setup failure]<br>cldeadlock<br>logfill<br>sc_truncate_lockorder_generated<br>sc_truncate</p>

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Error &#x26A0;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>1/470 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>ddl_analyze</p>

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Error &#x26A0;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>1/470 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>phys_rep</p>

**Review by chands10 — APPROVED** [review]

**Review by chands10 — APPROVED** [review]

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Error &#x26A0;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>1/470 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>pmux_sqlite_file_generated</p>

**Pull request merged**